### PR TITLE
Add stockpile-wrapper

### PR DESCRIPTION
**Pull request by rsevilla87**
Migrate stockpile-wrapper from Bohica.

Signed-off-by: Raul Sevilla <rsevilla@redhat.com>

### CODE REVIEW

**Review by jtaleric — COMMENTED** [review]
Nice! I like the idea overall!

File: Dockerfile [hunk 1]
```diff
@@ -1,15 +1,16 @@
 FROM registry.access.redhat.com/ubi8:latest
 
 RUN dnf install -y --nodocs git python3-pip hostname dmidecode pciutils procps-ng && dnf clean all
-RUN pip3 install --no-cache-dir  ansible elasticsearch elasticsearch-dsl openshift kubernetes redis
 RUN mkdir -p /backpack && chmod 770 /backpack
 ENV HOME /backpack
 WORKDIR /backpack
+COPY stockpile-wrapper/stockpile-wrapper.py stockpile-wrapper.py
+COPY stockpile-wrapper/requirements.txt requirements.txt
+RUN pip3 install -r requirements.txt
```
Comment (dry923) [inline Dockerfile:9]: I think you forgot to git add the requirements file ;-)

Comment (rsevilla87) [reply Dockerfile:9]: fixed!

**Review by dry923 — APPROVED** [review]
LGTM!

**Review by jtaleric — APPROVED** [review]
LGTM, even it it breaks things, death to Bohica!

**Pull request merged**